src/control/cmd/dmg/storage_query.go

```diff
@@ -71,7 +71,7 @@ type storageQueryCmd struct {
 
 type devHealthQueryCmd struct {
 	smdQueryCmd
-	UUID string `short:"u" long:"uuid" required:"1" description:"Device UUID"`
+	UUID string `short:"u" long:"uuid" description:"Device UUID. All devices queried if arg not set"`
```

Comment:
I don't know if the message is necessary here. Since the uuid is no longer required, I think the behavior can be inferred. Plus the command doesn't change state, so is safe for a user to experiment with.